util/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/AttributesUtil.java

```diff
@@ -42,6 +42,8 @@
 
   private static final int INSTANCES_LIMIT = 50000;
   private static final int STUDIES_SERIES_LIMIT = 5000;
+  
+  private static final String FUZZY_MATCHING = "true";
```

Comment:
Can we make this a boolean flag registered in https://github.com/GoogleCloudPlatform/healthcare-dicom-dicomweb-adapter/blob/master/import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/Flags.java so that users could enable or disable it if their backend doesn't support the fuzzy matching parameter?

Reply:
Yes that's a good idea, I'll make that change and will request your review.